lesson3/bigForm1/polymer/README.md

```diff
@@ -3,5 +3,7 @@ To install and run the Polymer checkout form:
 1. Install [npm](https://www.npmjs.com/), [bower](http://bower.io/) and [gulp](http://gulpjs.com/) if you haven't already
 2. Run `npm install`
 3. Run `bower install`
-4. Host over `localhost` and open `dest/index.html` in your browser
-5. As you make your edits to the files in `src/`, use `gulp watch` to watch for changes and rebuild to `dest/`.
\ No newline at end of file
+4. Run `gulp serve`
```

Comment:
I like that you added this. Can you make a change so that `gulp serve` runs the default task too?